push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushNotificationSender.java

```diff
@@ -27,6 +27,8 @@
  */
 public interface PushNotificationSender {
 
+    String AEROGEAR_PUSH_ID = "aerogear-push-id";
```

Comment:
as discussed on IRC we keep as it for now and we will see later if we can split the UnifiedMessage to hold "internal" information (https://issues.jboss.org/browse/AGPUSH-1381)